README.md

```diff
@@ -19,101 +19,4 @@ limitations under the License.
-
-- [Multi-Model Search](docs/config_search.md#multi-model-search-mode): **EARLY ACCESS** - Model Analyzer can help you
-  find the optimal settings when profiling multiple concurrent models, utilizing the [Quick Search](docs/config_search.md#quick-search-mode) algorithm
-
-### Other Features
-
-- [Detailed and summary reports](docs/report.md): Model Analyzer is able to generate
-  summarized and detailed reports that can help you better understand the trade-offs
-  between different model configurations that can be used for your model.
-
-- [QoS Constraints](docs/config.md#constraint): Constraints can help you
-  filter out the Model Analyzer results based on your QoS requirements. For
-  example, you can specify a latency budget to filter out model configurations
-  that do not satisfy the specified latency threshold.
-  <br><br>
-
-# Examples and Tutorials
-
-### **Single Model**
-
-See the [Single Model Quick Start](docs/quick_start.md) for a guide on how to use Model Analyzer to profile, analyze and report on a simple PyTorch model.
-
-### **Multi Model**
-
-See the [Multi-model Quick Start](docs/mm_quick_start.md) for a guide on how to use Model Analyzer to profile, analyze and report on two models running concurrently on the same GPU.
-
-### **Ensemble Model**
-
-See the [Ensemble Model Quick Start](docs/ensemble_quick_start.md) for a guide on how to use Model Analyzer to profile, analyze and report on a simple Ensemble model.
-
-### **BLS Model**
-
-See the [BLS Model Quick Start](docs/bls_quick_start.md) for a guide on how to use Model Analyzer to profile, analyze and report on a simple BLS model.
-<br><br>
-
-# Documentation
-
-- [Installation](docs/install.md)
-- [Model Analyzer CLI](docs/cli.md)
-- [Launch Modes](docs/launch_modes.md)
-- [Configuring Model Analyzer](docs/config.md)
-- [Model Analyzer Metrics](docs/metrics.md)
-- [Model Config Search](docs/config_search.md)
-- [Checkpointing](docs/checkpoints.md)
-- [Model Analyzer Reports](docs/report.md)
-- [Deployment with Kubernetes](docs/kubernetes_deploy.md)
-  <br><br>
-
-# Reporting problems, asking questions
-
-We appreciate any feedback, questions or bug reporting regarding this
-project. When help with code is needed, follow the process outlined in
-the Stack Overflow (https://stackoverflow.com/help/mcve)
-document. Ensure posted examples are:
-
-- minimal – use as little code as possible that still produces the
-  same problem
-
-- complete – provide all parts needed to reproduce the problem. Check
-  if you can strip external dependency and still show the problem. The
-  less time we spend on reproducing problems the more time we have to
-  fix it
-
-- verifiable – test the code you're about to provide to make sure it
-  reproduces the problem. Remove all other problems that are not
-  related to your request/question.
+> ##### THIS BRANCH IS UNDER ACTIVE DEVELOPMENT AND IS NOT READY FOR USE.
```

Comment:
Is this supposed to say this? This is the release branch, right? Why would it not be ready for use?

Reply:
@mc-nv friendly ping

Reply:
it's to avoid any issue raised against this branch buy customers.